PROCESSES.md

```diff
@@ -51,6 +64,30 @@ maintainers:
   url:    # The URL of the GitHub profile, using the format https://github.com/<username>. Required. 
 ```
 
+Each maintainer should have a GitHub account
+and part of the `@prometheus-community/helm-charts-maintainers` GitHub group and member of `@prometheus-community` organization.
+
+## Removing Chart Maintainers
+
+If a chart maintainer is no longer active, not part of the `@prometheus-community` organization,
+or has left the project, they should be removed from the `Chart.yaml` file.
+To do this,
+update the `maintainers` section in `Chart.yaml` and remove them from the [CODEOWNERS](./.github/CODEOWNERS) file.
+
+## Adding a New Chart
+
+This process is not yet documented.
+
+## Deprecating a Chart
+
+A chart should be deprecated if it is no longer maintained, has no active maintainer, or should not be used anymore.
```

Comment:
For me it's a bit unclear, what "is no longer maintained" means?
Some charts might be feature complete and it might be enough to get renovate updates of the docker image.

Reply:
Good point. There are two proposals to define this:

* The upstream project is being deprecated, so we should deprecate the chart as well.
* Not responding to PRs can be considered a sign that the project is no longer maintained.

Reply:
Point 1 would be a reason for me to deprecate a chart.

Point 2 would be up for discussion. 
The chart would likely still work even if no new features are added / reviewed as long as updates are done by renovate.